deploy/0060_deploy_gems.ts

```diff
@@ -6,13 +6,13 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   const {deployments, getNamedAccounts} = hre;
   const {deploy} = deployments;
 
-  const {deployer} = await getNamedAccounts();
-  for (const gemName of gems) {
-    await deploy(`Gem_${gemName}`, {
-      contract: 'ERC20Token',
+  const {gemMinter, deployer} = await getNamedAccounts();
+  for (const gem of gems) {
+    await deploy(`Gem_${gem.name}`, {
+      contract: 'Gem',
       from: deployer,
       log: true,
-      args: [gemName, gemName, deployer],
+      args: [gem.name, gem.name, gemMinter, gem.gemId],
```

Comment:
gem name and symbol should be different
name : `Sandbox's <symbol> Gems`
symbol: `<symbol>`

example : 
name: Sandbox's LUCK Gems
symbol: LUCK